hack/util.sh

```diff
@@ -8,235 +8,240 @@ TIME_MIN=$((60 * $TIME_SEC))
 # setup_env_vars exports all the necessary environment variables for configuring and
 # starting OS server.
 function setup_env_vars {
-	# set path so OpenShift is available
-	GO_OUT="${OS_ROOT}/_output/local/bin/$(os::util::host_platform)"
-	export PATH="${GO_OUT}:${PATH}"
-
-	export ETCD_PORT="${ETCD_PORT:-4001}"
-	export ETCD_PEER_PORT="${ETCD_PEER_PORT:-7001}"
-	export API_BIND_HOST="${API_BIND_HOST:-$(openshift start --print-ip)}"
-	export API_HOST="${API_HOST:-${API_BIND_HOST}}"
-	export API_PORT="${API_PORT:-8443}"
-	export LOG_DIR="${LOG_DIR:-${BASETMPDIR}/logs}"
-	export ETCD_DATA_DIR="${BASETMPDIR}/etcd"
-	export VOLUME_DIR="${BASETMPDIR}/volumes"
-	export FAKE_HOME_DIR="${BASETMPDIR}/openshift.local.home"
-	export API_SCHEME="${API_SCHEME:-https}"
-	export MASTER_ADDR="${API_SCHEME}://${API_HOST}:${API_PORT}"
-	export PUBLIC_MASTER_HOST="${PUBLIC_MASTER_HOST:-${API_HOST}}"
-	export KUBELET_SCHEME="${KUBELET_SCHEME:-https}"
-	export KUBELET_BIND_HOST="${KUBELET_BIND_HOST:-$(openshift start --print-ip)}"
-	export KUBELET_HOST="${KUBELET_HOST:-${KUBELET_BIND_HOST}}"
-	export KUBELET_PORT="${KUBELET_PORT:-10250}"
-	export SERVER_CONFIG_DIR="${BASETMPDIR}/openshift.local.config"
-	export MASTER_CONFIG_DIR="${SERVER_CONFIG_DIR}/master"
-	export NODE_CONFIG_DIR="${SERVER_CONFIG_DIR}/node-${KUBELET_HOST}"
-	export ARTIFACT_DIR="${ARTIFACT_DIR:-${BASETMPDIR}/artifacts}"
-	if [ -z ${SUDO+x} ]; then
-		export SUDO="${SUDO:-1}"
-	fi
-
-	# Use either the latest release built images, or latest.
-	if [[ -z "${USE_IMAGES-}" ]]; then
-		IMAGES='openshift/origin-${component}:latest'
-		export TAG=latest
-		export USE_IMAGES=${IMAGES}
-		if [[ -e "${OS_ROOT}/_output/local/releases/.commit" ]]; then
-			COMMIT="$(cat "${OS_ROOT}/_output/local/releases/.commit")"
-			IMAGES="openshift/origin-\${component}:${COMMIT}"
-			export TAG=${COMMIT}
-			export USE_IMAGES=${IMAGES}
-		fi
-	fi
-
-	if [[ "${API_SCHEME}" == "https" ]]; then
-		export CURL_CA_BUNDLE="${MASTER_CONFIG_DIR}/ca.crt"
-		export CURL_CERT="${MASTER_CONFIG_DIR}/admin.crt"
-		export CURL_KEY="${MASTER_CONFIG_DIR}/admin.key"
-	fi
-
-	# change the location of $HOME so no one does anything naughty
-	export HOME="${FAKE_HOME_DIR}"
+  # set path so OpenShift is available
```

Comment:
> everytime I see tabs instead of spaces my vim and my eyes hurt and I have compulsive need to do :retab ;-) will revert

You should be able to set the spacing.  @csrwng knows the magic incantation